### PR TITLE
Task TwigCs

### PR DESCRIPTION
**Pull request by oallain**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Documented?   | yes
| Fixed tickets | 

Check [Twig](https://twig.symfony.com/) coding standard based on [Allocine/TwigCs](https://github.com/allocine/twigcs) .
At the moment the only available standard is the [official one from twig](http://twig.sensiolabs.org/doc/coding_standards.html).

# New Task Checklist:

- [x] Is the README.md file updated?
- [x] Are the dependencies added to the composer.json suggestions?
- [x] Is the doc/tasks.md file updated?
- [x] Are the task parameters documented?
- [x] Is the task registered in the tasks.yml file?
- [x] Does the task contains phpspec tests?
- [x] Is the configuration having logical allowed types?
- [x] Does the task run in the correct context?
- [x] Is the `run()` method readable?
- [x] Is the `run()` method using the configuration correctly?
- [x] Are all CI services returning green?

### CODE REVIEW

**Comment by oallain** [comment]
Hello @veewee @Landerstraeten ,

What do you think about this ?

File: doc/tasks/twigcs.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+parameters:
+    tasks:
+        twigcs:
+            path: '.'
+            severity: 'warning'
+            ruleset: 'Allocine\Twigcs\Ruleset\Official'
+            triggered_by: ['twig']
+```
+
+**path**
```
Comment (Landerstraeten) [inline doc/tasks/twigcs.md]: @veewee, rfc:
```
**path**

*Default: null*

By default `.` wil be used.
You can specify an alternate location by changing this option.
```

Comment (oallain) [reply doc/tasks/twigcs.md]: Hello @Landerstraeten,

Thank you for this feedback, i changed it :smile:

**Comment by veewee** [comment]
Thanks!

**Comment by oallain** [comment]
> Thanks!

@veewee ,do you know when Milestone 0.15.1 will be tag/released ?

**Comment by Landerstraeten** [comment]
@oallain Hopefully tomorrow

**Comment by oallain** [comment]
Greate !

**Comment by oallain** [comment]
@Landerstraeten Before creating the tag, I think we have to solve this issue, it is blocking for instalation.
https://github.com/phpro/grumphp/issues/633

**Comment by oallain** [comment]
Apologies, this is fix on master :D

**Pull request merged**